src/AutoRest.CSharp/LowLevel/AutoRest/LowLevelTarget.cs

```diff
@@ -111,6 +112,20 @@ public static async Task ExecuteAsync(GeneratedCodeWorkspace project, InputNames
             contextWriterInstance.Write(contextWriter);
             project.AddGeneratedFile($"Models/{ModelReaderWriterContextWriter.Name}.cs", contextWriter.ToString());
 
+            // Generate ModelReaderWriterBuildableAttributes if using ModelReaderWriter
+            if (Configuration.UseModelReaderWriter)
+            {
+                var buildableAttributesWriter = new ModelReaderWriterBuildableAttributesWriter();
+                var buildableTypes = buildableAttributesWriter.CollectBuildableTypes(library.AllModels);
+                
+                if (buildableTypes.Any())
+                {
+                    var attributesWriter = new CodeWriter();
+                    buildableAttributesWriter.WriteModelReaderWriterBuildableAttributes(attributesWriter, buildableTypes);
+                    project.AddGeneratedFile("ModelReaderWriterBuildableAttributes.cs", attributesWriter.ToString());
```

Comment:
This should not be its own file these attributes should be added to the Context class that is added into the generated folder today.